src/components/AmsterdamLogo/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { LogoShort } from '@datapunt/asc-assets';
+import styled, { css } from 'styled-components';
+import { breakpoint, themeSpacing, themeColor } from '@datapunt/asc-ui';
+
+export const srOnlyStyle = () => ({ srOnly }) =>
```

Comment:
What is `srOnly` and why should there be a check for it when the prop `srOnly` is always true?

Reply:
The screen readers variant of the logo. I removed that.